package.json

```diff
@@ -15,6 +15,7 @@
     "env-cmd": "^10.1.0",
     "file-saver": "^2.0.5",
     "html-to-image": "^1.6.2",
```

Comment:
do we still need `html-to-image`?

Reply:
@yuvalbl no, it's not needed. I removed it